lib/apipie/param_description.rb

```diff
@@ -101,7 +101,7 @@ def initialize(method_description, name, validator, desc_or_options = nil, optio
     end
 
     def from_concern?
-      method_description.from_concern? || @from_concern
+      method_description.from_concern? || (defined?(@from_concern) && @from_concern)
```

Comment:
I think here we can update lines

https://github.com/Apipie/apipie-rails/blob/b0bc6d60dbe2cedfa155ba4939f71b1943766509/lib/apipie/param_description.rb#L56-L58

from 
```ruby
if @options[:param_group]
  @from_concern = @options[:param_group][:from_concern]
end
```

to 
```ruby
@from_concern = @options.dig(:param_group, :from_concern)
```